### PR TITLE
Add unit test for` ProcessVesting`: empty vester balance

### PR DESCRIPTION
**Pull request by teddyding**

### CODE REVIEW

File: protocol/x/vest/keeper/keeper_test.go [hunk 1]
```diff
@@ -164,6 +164,20 @@ func TestProcessVesting(t *testing.T) {
 			expectedTreasuryBalance: sdk.NewInt(3_007),
 			expectedVesterBalance:   sdk.NewInt(2_001_993),
 		},
+		"vesting in progress,  start_time < prev_block_time < block_time < end_time, vester has empty balance": {
```
Comment (ttl33) [inline protocol/x/vest/keeper/keeper_test.go:167]:
```suggestion
		"vesting in progress: expect 0 balance,  start_time < prev_block_time < block_time < end_time, vester has empty balance": {
```

File: protocol/x/vest/keeper/keeper_test.go [hunk 2]
```diff
@@ -193,6 +207,20 @@ func TestProcessVesting(t *testing.T) {
 			expectedTreasuryBalance: sdk.NewInt(668),
 			expectedVesterBalance:   sdk.NewInt(2_004_332),
 		},
+		"vesting just started, prev_block_time < start_time < block_time < end_time, vester has empty balance": {
```
Comment (ttl33) [inline protocol/x/vest/keeper/keeper_test.go:210]:
```suggestion
		"vesting just started: expect 0 balance, prev_block_time < start_time < block_time < end_time, vester has empty balance": {
```

**Review by ttl33 — APPROVED** [review]

File: protocol/x/vest/keeper/keeper_test.go (hunk 1 above)
Comment (teddyding) [reply protocol/x/vest/keeper/keeper_test.go:167]: Personally, I prefer `vesting in progress,  start_time < prev_block_time < block_time < end_time, vester has empty balance` is pretty clear and more concise

File: protocol/x/vest/keeper/keeper_test.go (hunk 2 above)
Comment (teddyding) [reply protocol/x/vest/keeper/keeper_test.go:210]: Same as above

**Pull request merged**